modules/videoio/src/cap_msmf.cpp

```diff
@@ -2469,6 +2469,10 @@ const GUID CvVideoWriter_MSMF::FourCC2GUID(int fourcc)
 #endif
         case CV_FOURCC_MACRO('H', '2', '6', '4'):
                 return MFVideoFormat_H264; break;
+        case CV_FOURCC_MACRO('H', '2', '6', '5'):
+                return MFVideoFormat_H265;  break;
+        case CV_FOURCC_MACRO('H', 'E', 'V', 'C'):
+                return MFVideoFormat_HEVC;  break;            
```

Comment:
There is a compilation problem with some Windows SDK versions.

> `error C2065: 'MFVideoFormat_H265': undeclared identifier`

Need a proper `#if` check/guard for using of these values.

Reply:
Will do. Thanks.